src/resources/i18n/translations/en.json

```diff
@@ -488,10 +488,10 @@
   "No Wifi Connection": "No Wi-Fi connection",
   "No Wifi Only": "No, Wi-Fi only",
   "Notifications": "Notifications",
-  "Notifications UP help text": "In order to receive push notifications using UnifiedPush, you will need a distributor app installed on your device. After it is installed, then select a distributor from the dropdown above. A popular option for a UP distributor app is called ntfy. Also, you should disable \"battery optimization\" for your distributor app, to make sure your phone is always listening for notifications.",
+  "Notifications UP help text": "Selcet an installed provider (like ntfy) from the dropdown menu to use UnifiedPush. Turn off battery optimization for it to not miss any notifications.",
```

Comment:
Couldn't find this in the menus. Where is it?

Reply:
The helper text is supposed to be revealed when the "UnifiedPush notifications" toggle is enabled. Maybe it should be showing even when it is not toggled yet though...

Reply:
Help or no help, I can't find it.
"Enable UnifiedPush notifications" isn't very descriptive.
"Immediate notifications" and then an explanation of how that is done through UP?

Reply:
@comradekingu oh my bad 🤦‍♂️ you would have to have the F-Droid version running locally. I'm publishing the new version to the F-Droid Store tonight, and it takes about a week before it is downloadable.

In the meantime, if you'd like to download the 4.13.1 beta before that, it can be found at https://files.podverse.fm.